src/applications/sign-in-changes/containers/InterstitialChanges.jsx

```diff
@@ -1,17 +1,28 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useState, useEffect } from 'react';
 import { VaLink } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
-import { selectProfile } from 'platform/user/selectors';
+import { apiRequest } from '@department-of-veterans-affairs/platform-utilities/exports';
 import { AUTHN_SETTINGS } from '@department-of-veterans-affairs/platform-user/exports';
 import CreateAccount from '../components/CreateAccount';
 import AccountSwitch from '../components/AccountSwitch';
 
 export default function InterstitialChanges() {
-  const userHasLogingov =
-    useSelector(state => selectProfile(state)?.logingovUuid) !== null;
-  const userHasIdme =
-    useSelector(state => selectProfile(state)?.idmeUuid) !== null;
-  const showAccount = userHasLogingov || userHasIdme;
+  const [userEmails, setUserEmails] = useState({});
+
+  useEffect(() => {
+    apiRequest('/sign-in-changes-reminder', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
+      .then(response => response.json())
+      .then(data => {
+        setUserEmails(data.user?.credential_type?.email || {});
+      })
```

Comment:
`apiRequest` already does the `.json()` call so probably only need a single `.then`